Sentry.xcodeproj/project.pbxproj

```diff
@@ -1553,8 +1553,8 @@
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "fastlane oc_lint\noclint-json-compilation-database -- -report-type xcode\n";
+			shellPath = /bin/zsh;
+			shellScript = "source ~/.zshrc\ncd ${SRCROOT}\nfastlane oc_lint\nmaxPriority=15000\n# ignore SentryCrash\noclint-json-compilation-database -exclude SentryCrash -- -report-type xcode -max-priority-1=$maxPriority -max-priority-2=$maxPriority -max-priority-3=$maxPriority\n";
```

Comment:
`source ~/.zshrc\ncd` this will most likely not work 😅 
What is need in this config to make this work?

Reply:
true. will check if needed, most likely drop it. will change shellPath to default `/bin/sh` as well.

Reply:
@HazAT works after changes